lib/circuitry.rb

```diff
@@ -1,3 +1,4 @@
+require File.join(File.dirname(__FILE__), 'railtie.rb') if defined?(Rails) && Rails::VERSION::MAJOR >= 3
```

Comment:
With the other change I recommended, this just becomes `require 'circuitry/railtie'`